apis/cr/v1/cluster.go

```diff
@@ -88,6 +91,14 @@ type PgclusterStatus struct {
 // swagger:ignore
 type PgclusterState string
 
+// PodAntiAffinityType defines the different types of type of anti-affinity rules applied to pg
+// clusters when utilizing the default pod anti-affinity rules provided by the PGO, which are
```

Comment:
I'd recommend referring to it as the "PostgreSQL Operator" as "PGO" is more likely to be grepped when looking up things related to the CLI

Reply:
Changed to "PostgreSQL Operator".